litmus/director/openebs-control-plane-test/test.yml

```diff
@@ -0,0 +1,264 @@
+---
+- hosts: localhost
```

Comment:
Need to explain in code comments the role of hosts & connections.
Need to explain if the code is run from within a container or outside a container & so on.